swap_meet/clothing.py

```diff
@@ -1,2 +1,19 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    '''
+    Class representing a clothing item to be traded
+    Child of Item class
+
+    Methods:
+        condition_description() from Item class
+
+    Variables:
+        category: category of clothing default is Clothing
+        condition: condition of an item from 0-5
+    '''
+    def __init__(self, category="Clothing", condition=0):
+        super().__init__(category, condition)
```

Comment:
With this approach, we are allowing instances of Clothing to have categories other than `Clothing`. To ensure that all instances have the same category of `Clothing` and prevent any caller from changing it, we will need to remove `category` from the constructor. 

Instead, we should pass the category value to the parent's constructor like so:

```python
    def __init__(self, condition=0):
        super().__init__(category="Clothing", condition)
```